tests/test_det_employment_solutions_corp.py

```diff
@@ -0,0 +1,94 @@
+from datetime import datetime
+from operator import itemgetter
+from os.path import dirname, join
+
+import pytest  # noqa
+from city_scrapers_core.constants import COMMITTEE, BOARD, PASSED
+from city_scrapers_core.utils import file_response
+from freezegun import freeze_time
+
+from city_scrapers.spiders.det_employment_solutions_corp import DetEmploymentSolutionsCorpSpider
+
+test_pdf_response = file_response(
+    join(dirname(__file__), "files", "det_employment_solutions_corp.pdf"),
+    url=(
+        "https://www.descmiworks.com/wp-content/uploads/2019-DESC-Board-And-Committee-Calendar-Amended-08-22-2019.pdf"  # noqa
+    ),
+    mode="rb",
+)
+test_response = file_response(
+    join(dirname(__file__), "files", "det_employment_solutions_corp.html"),
+    url="https://www.descmiworks.com/about-us/public-meetings/",
+)
+spider = DetEmploymentSolutionsCorpSpider()
+
+freezer = freeze_time("2019-07-20")
+freezer.start()
+
+spider._parse_schedule_pdf(test_pdf_response)
+parsed_items = sorted(
+    [item for item in spider.parse(test_response)],
```

Comment:
It looks like changing this to use `_parse_documents` should fix your problem. `parse` is yielding another Request object, so you can cut out some steps that the actual scraper is doing

Reply:
Okay, thanks!